ReviewMonitor/Apps List View Controller/App Detail View Controller/MetaViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  MetaViewController.swift
+//  ReviewMonitor
+//
+//  Created by Eliott Hauteclair on 3/01/18.
+//  Copyright © 2018 Tayal, Rishabh. All rights reserved.
+//
+
+import UIKit
+
+protocol MetaViewControllerDelegate: class {
+    func metaControllerDidDismiss()
+}
+
+class MetaViewController: UIViewController {
+
+    weak var delegate: AccountsViewControllerDelegate?
+
+    var m = Meta()
```

Comment:
Let's name this vocal. Can we change this to `var metadata = Meta()`?

Reply:
Yes